File: EMI/Faraday-BE-gRPC/Models/Customer.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Faraday_BE_gRPC.Models
{
    public class Customer
    {

        public Customer(int id, string firstName, string  lastName, string gender, string driversLicense)
        {
            Id = id;
            FirstName = firstName;
            LastName = lastName;
            Gender = gender;
            DriversLicense = driversLicense;
        }

        public Customer( string firstName, string lastName, string gender, string driversLicense)
        {
            
            FirstName = firstName;
            LastName = lastName;
            Gender = gender;
            DriversLicense = driversLicense;
        }

        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Gender { get; set; }
        public string DriversLicense { get; set; }
    }
}

File: Exam-Project/Spring Workspace/CustomerService/src/main/java/com/mwe/CustomerService/Models/Customer.java
package com.mwe.CustomerService.Models;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

@Entity
@Table(name = "`Customers`")
public class Customer {
	
	@Id
	@GeneratedValue(strategy = GenerationType.AUTO)
	@Column(name="id")
	private Integer id;
	@Column(name="first_name")
	private String firstName;
	@Column(name="last_name")
	private String lastName;
	@Column(name="gender")
	private String gender;
	@Column(name="drivers_license")
	private String driversLicense;
	
	public Customer() {
		
	}

	public Customer(String firstName, String lastName, String gender, String driversLicense) {
		super();
		this.firstName = firstName;
		this.lastName = lastName;
		this.gender = gender;
		this.driversLicense = driversLicense;
	}



	public String getFristName() {
		return firstName;
	}

	public void setFristName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public String getGender() {
		return gender;
	}

	public void setGender(String gender) {
		this.gender = gender;
	}

	public String getDriversLicense() {
		return driversLicense;
	}

	public void setDriversLicense(String driversLicense) {
		this.driversLicense = driversLicense;
	}
	
	
}
